DC/6.4.0/formats/glb_preview.dcl

```diff
@@ -41,3 +41,8 @@ resource format glb_preview {
     system = true
 }
 
+patch format glb {
+    target = resource.format.glb_preview
+    name = 'GLB'
+}
```

Comment:
Resource definitions can not be patched 😉 
Patches are defined to allow us to patch properties of resources that are defined in lower-level configuration layers.

Take a look at my comment [here](https://keyshot.atlassian.net/browse/DAM-8500?focusedCommentId=377639).
The name of the `glb_preview` resource needs to be changed directly in the resource definition. Furthermore, we need to add a Configuration Management resource migration to ensure that this doesn't break anything.
Let me know if you have any questions related to this 😉